locations/spiders/audika_gb.py

```
@@ -0,0 +1,45 @@
+        item["name"] = "Audika"
+        centerid = re.search(
+            r"([0-9]+)$",
+            response.xpath("//a[starts-with(@href,'/book-appointment/online-booking?centerid=')]/@href").get(),
+        )
+        if centerid:
+            item["extras"]["center_id"] = centerid.group(1)
+        item["addr_full"] = item.pop("street_address")
+        if item.get("email") and "info@audika.co.uk" in item["email"]:
+            del item["email"]
+        item["opening_hours"] = OpeningHours()
+        for str in raw_data.get("openingHours"):
```

Ruff (0.16.2)

[error] Variable `str` is shadowing a Python builtin

(A001)